src/adagrams.js

```diff
@@ -1,15 +1,145 @@
+const letterPool = [
```

Comment:
Just a reminder that when we create global javascript constants, we use uppercase snake case names like LETTER_POOL or LETTER_VALUES!